api/test/pact/pact-tests/wa2/taskRoles.spec.ts

```diff
@@ -118,8 +99,8 @@ describe('Task management api, task roles', () => {
 
 function assertResponses(dto: any) {
   console.log(JSON.stringify(dto));
-  expect(dto[0].role_category).to.be.equal('LEGAL_OPERATIONS');
-  expect(dto[0].role_name).to.be.equal('case-worker');
-  expect(dto[0].permissions).to.include.members(['OWN', 'EXECUTE', 'READ', 'MANAGE', 'CANCEL']);
-  expect(dto[0].authorisations).to.include.members(['IAC', 'SSCS']);
+  // expect(dto[0].role_category).to.be.equal('LEGAL_OPERATIONS');
```

Comment:
This is a bit dodgy. Are there no assertions at all to be made? Should be a ticket to make this test meaningful again